src/main/java/aztech/modern_industrialization/datagen/tag/TagsToGenerate.java

```diff
@@ -33,6 +34,7 @@
 public class TagsToGenerate {
 
     static final Map<TagKey<Item>, List<ItemLike>> tagToItemMap = new HashMap<>();
+    static final Map<ItemLike, SortOrder> itemSortOrderMap = new HashMap<>();
```

Comment:
The key should be Item not ItemLike, otherwise it might not be found in the map if queried with a different ItemLike that has the same Item.